src/components/line-chart/line-chart.component.ts

```diff
@@ -69,21 +75,13 @@ export class LineChartComponent implements RenderOptions, OnChanges, OnInit, OnD
   }
 
   ngOnInit() {
-    this.dataService.observeTimeSeries(this.queryOptions$)
+    this.queryOptions$
       .pipe(takeUntil(this.destroy$))
-      // TODO: the pipe below will be removed once line chart supports rendering multiple measures
-      .pipe(map(rows => {
-        this.activeDatum$.next(null);
-        if (!rows.length) {
-          return [];
-        }
-        const [firstMeasureName] = Object.keys(rows[0].values);
-        return rows.map(row => ({
-          date: row.categories.date,
-          value: row.values[firstMeasureName],
-        }));
+      .pipe(map(queryOption => {
+        this.activePoint$.next(null);
+        return this.meta.query(queryOption)[0];
```

Comment:
Could you add a comment explaining the significance of index 0 here?

Reply:
Sure, I will add a comment to it.

It is just a workaround. `LineChartComponent` should render multiple lines when there are more than one datum in the array, but it is not able to do that now. This feature will be supported in #45 .